.eslintrc

```diff
@@ -2,6 +2,7 @@
   "extends": "airbnb",
   "parser": "babel-eslint",
   "rules": {
+    "arrow-parens": [2, "as-needed"],
```

Comment:
Could you modify this file in the tools repository first, please ? https://github.com/Talend/tools/blob/master/tools-javascript/.eslintrc

Reply:
Sure, when it's ok here :)

Reply:
It's available [here](https://github.com/Talend/tools/pull/53)